man/data.table.Rd

```diff
@@ -19,6 +19,7 @@
 }
 \usage{
 data.table(\dots, keep.rownames=FALSE, check.names=FALSE, key=NULL, stringsAsFactors=FALSE)
+rowwiseDT(..., key=NULL)
```

Comment:
`rowwiseDT` is not present as an alias in the Rd file. Furthermore, `...` should be `\dots`.

IMO `rowwiseDT` should be documented as a separate function and cross-linked (in see also and in the examples section) in the documentation of `data.table()`.